tt_eager/tt_dnn/queue_api.hpp

```diff
@@ -0,0 +1,79 @@
+void EnqueueAllocateDeviceBuffer(Queue&, DeviceBuffer&);
+void EnqueueAllocateHostBuffer(Queue&, HostBuffer&);
+
+
+void EnqueueDeallocate(Queue&, Tensor&);
+void EnqueueReallocate(Queue&, Tensor&);
+
+
+void EnqueueHostToDeviceTransfer(Queue&, Tensor& src, Tensor& dst);
+void EnqueueDeviceToHostTransfer(Queue&, Tensor& src, Tensor& dst);
+
+
+void QueueRecordEvent(Queue&, Event&);
+void QueueWaitForEvent(Queue&, Event&);
+void EventSynchronize(Event&);
+
```

Comment:
I think this one is missing:
_void QueueSynchronize(Queue&);_

Reply:
Yes I agree. This new API just waits until all jobs in Queue are complete, correct?

Reply:
Correct.